x-pack/plugin/spatial/build.gradle

```diff
@@ -20,6 +20,8 @@ dependencies {
   testImplementation project(path: ':modules:percolator')
   testImplementation project(path: xpackModule('vector-tile'))
   testImplementation project(path: ':modules:aggregations')
+  testImplementation project(path: xpackModule('esql'))
```

Comment:
I remember previously when we had code that depended on both xpack-spatial and xpack-esql, we moved the spatial classes to server or lib, so we could have the code in ESQL. We did consider adding a dependency on spatial within esql (so esql could use spatial code), but we did not consider doing the reverse, adding a dependency on esql from spatial. Is this the right move?